roles/ssh_jumper/tasks/cleanup.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Remove Include cifw_ssh_config.d
```

Comment:
I'd almost expect some `delegate_to: localhost` there?

Reply:
I think that should be up to the caller?
We do put these jumper configs on all the nodes? controller, compute, networker.

Reply:
hmm right. ok.
So like now, we need to run twice, on ansible controller (usually laptop) and hypervisor, with appropriate delegations.